vscode-car-plugin/src/main/java/org/wso2/maven/datamapper/Constants.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2024, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Shall we change Inc to LLC? Let's change in other relevant areas as well.

Reply:
Changed with 4403dbe2478779614754571895fd91bedd953b10